price-feeder.example.toml

```diff
@@ -212,6 +224,31 @@ providers = [
 ]
 quote = "OSMO"
 
+[[currency_pairs]]
+base = "AKT"
+quote = "USDT"
+providers = [ "huobi", "gate",]
+
+[[currency_pairs]]
+base = "AKT"
+quote = "USD"
+providers = [ "kraken", "crypto",]
+
+[[currency_pairs]]
+base = "JUNO"
+quote = "USDT"
+providers = [ "bitget", "mexc",]
+
+[[currency_pairs]]
+base = "JUNO"
+quote = "USD"
+providers = [ "kraken",]
+
+[[currency_pairs]]
+base = "LUNA"
+quote = "USDT"
+providers = [ "okx", "gate", "huobi", "bitget",]
```

Comment:
Let's add osmosis to each of these. Might need to deploy a config change to staging

Reply:
Osmosis PR for JUNO: https://github.com/umee-network/osmosis-api/pull/129. AKT and LUNA have good enough volume on the current providers.

Reply:
merged in 👍 let's do another local test, update canon, and then share the updated config